tests/org.eclipse.n4js.n4ide.spec.tests/xpect-test/Ch03_02_02_08__QuickFix_Type_Issues/CONVERT_JAVA_TO_COLON_STYLE.n4js.xt

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
I think we named the quickfix tests after the issue, this is why some tests have upper case names. In your case, that would be TYS_INVALID_TYPE_SYNTAX.n4js.xt